js/views/homeVw.js

```diff
@@ -421,8 +439,26 @@ module.exports = Backbone.View.extend({
     this.$el.find('.js-homeSearchItemsClear').addClass('hide');
   },
 
+  setListingsBlockedCount: function(count) {
+    var text = '';
+
+    count = count || 0;
+    if (this.blockedItemCount === count) return;
+    this._blockedItemCount = count;
+    
+    if (count) {
+      text = count + ' blocked item' + (count !== 1 ? 's' : '') + ' not shown';
+    }
+
+    this.$listingsBlockedCount = this.$listingsBlockedCount || this.$('.homeGridItems .js-blocked-listings-count');
```

Comment:
yeah, this is kind of a lazy way. Typically, frequently used selectors are cached in render, but since that's not really being done in this view and if you have a selector that's only used in one place, you could probably get away with this pattern.

If this selector were used in many places, you probably wouldn't want to keep repeating this line.